### DISCUSSION TITLE
Advanced database configuration options [May 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5654] [Discord Threads]
> 🙋‍♂️ **MyWay**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Few of our users ended up messing database fields & hence these features are hidden for advanced users under and easter egg menu.   
#### Details here   
#### https://docs.nocodb.com/setup-and-usages/table-operations#column-edit  
#  
> 🙋‍♂️ **MyWay**  
#### Thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1094714777814695966)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
